dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -69,35 +89,197 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     }
 }
 
+impl From<&Arc<Log>> for StandingOrder {
+    fn from(log: &Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        info!("Parsing StandingOrder from bytes. {} bytes. {:?}", bytes.len(), bytes);
+        
+        // Get basic data from event
+        let batch_index = U256::pop_from_log_data(&mut bytes);
+        let valid_from_auction_id = U256::pop_from_log_data(&mut bytes);
+        let account_id = u16::pop_from_log_data(&mut bytes);
+
+        let bytes_init = u8::pop_from_log_data(&mut bytes) as usize;
+        let byte_size = u8::pop_from_log_data(&mut bytes) as usize;
+
+        info!("Extracting packed order. Bytes: {}-{}", bytes_init, bytes_init + byte_size);
+        let packed_orders_bytes = &bytes[0..byte_size];
+        info!("Parsing orders from packedOrders. {} bytes. {:?}", packed_orders_bytes.len(), packed_orders_bytes);
+        assert!(packed_orders_bytes.len() % 26 == 0, "Each order should be packed in 26 bytes");
+                
+        // Extract packed order info
+        let orders: Vec<models::Order> = packed_orders_bytes
+            .chunks(26)
+            .map(|chunk| models::Order::from_encoded_order(account_id, chunk.to_vec()))
+            .collect();
+
+        StandingOrder {
+            account_id,
+            batch_index,
+            valid_from_auction_id,
+            orders
+        }
+    }
+}
+
+
+impl From<Entity> for StandingOrder {
```

Comment:
I'd argue that the signature should look more like

```rs
impl From<(Entity, Vec<Entity>)> ...
impl Into<(Entity, Vec<Entity>)> ...
```

where the first entity is of type StandingOrder and the vector is the orders associated with the StandingOrder.

It would be nice if we got a coherent entity set back from the into method instead of relying on the handler to mutate the resulting entity with the order IDs.

Reply:
As we agreed later, to avoid having to include ids in the model and adapting the code (including the mongo impl), I've implemented just the `From<(Entity, Vec<Entity>)>` for now

Reply:
I'm confused not sure which other into you are referring :) 

Also, super weird the order of the comments here, hahaha:

![image](https://user-images.githubusercontent.com/2352112/63107115-a36e1680-bf84-11e9-92f9-a9093a15ef26.png)